### DISCUSSION TITLE
Comparison with other AMB frameworks

### DISCUSSION THREAD

**Discussion by davpaez** [JoelForamitti/agentpy#18] [Q&A] [answered]
Is there any article or resource with a feature comparison with other frameworks AMB frameworks? Some I can think of: Mesa, Netlogo, Repast, Mason.

Thank you!

**Comment by jofmi** [comment, marked as answer]
Hi @davpaez, I have started a comparison page here: https://agentpy.readthedocs.io/en/latest/comparison.html
It could probably still be improved a lot, and focuses mainly on AgentPy vs. Mesa. 
Let me know if it is (or isn't) useful!

**Reply by davpaez** [reply]
Hi @JoelForamitti. It is very useful. Sure, it can be expanded further, but it's a great start 😎️ It helped me to understand key differences at a high level and also in terms of design patterns and syntax.
Thank you so much!

**Reply by jamesdamillington** [reply]
@JoelForamitti One day if I have time I can try to provide some NetLogo code that reproduces the 'wealth transfer' simple model on [the comparison page](https://agentpy.readthedocs.io/en/latest/comparison.html). And maybe also suggest entries for a NetLogo column in the comparison table too. If you think it is worthwhile.

**Reply by jofmi** [reply]
@jamesdamillington that would be fantastic!

**Reply by jamesdamillington** [reply]
@jofmi  I just found this old thread. We're years on, but [here](https://modelingcommons.org/browse/one_model/7626#model_tabs_browse_procedures) is the equivalent model code for NetLogo.

Run the model via the 'Run in NetLogo Web' tab -> click blue Setup button, then blue Go button, and the output will be in the Command Center (view via dropdown). Change parameters using the sliders (click setup before re-running with go button). Experiments can be run using NetLogo's 'BehaviorSpace'

**Reply by jofmi** [reply]
Oh this is great, thank you! I will put on my todo list to update the comparison, also with the new mesa version - maybe we could put it in a blog post or something?
